data/brands/amenity/fuel.json

```diff
@@ -49,6 +49,21 @@
         "name": "7-Eleven"
       }
     },
+    {
+      "displayName": "Mapco",
+      "locationSet": {
+        "include": ["001"],
+        "exclude": ["pl"]
+      },
+      "matchNames": ["mapco", "MAPCO"],
+      "tags": {
+        "amenity": "fuel",
+        "brand": "Mapco",
+        "brand:wikidata": "Q107589462",
+        "brand:wikipedia": "en:Mapco",
```

Comment:
There doesn't seem to be an article for Mapco in the English Wikipedia, only Wikidata tags are required. You could make a Wikipedia article, but note that it's more difficult than creating a Wikidata page.